docker/docker-compose.dev.yml

```diff
@@ -0,0 +1,47 @@
+# Docker Compose file for development
+version: "2"
+services:
+
+  db:
+    image: postgres:9.5.4
+    volumes:
+      - ../data/pgdata:/var/lib/postgresql/data/pgdata
+    environment:
+      PGDATA: /var/lib/postgresql/data/pgdata
+    ports:
+      - "15432:5432"
+
+  webserver:
+    build:
+      context: ..
+      dockerfile: ./docker/Dockerfile.dev
+    volumes:
+      - ../:/code
+      - ../data/app:/data
+    ports:
+      - "8080:8080"
```

Comment:
We use the 80 port in CB and LB.

Reply:
My preference for development is to use a non-standard port, in case 80 is already used. It's only a small difference so I'll leave it for now